microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/v20/MicroProfileOpenApi20Test.java

```diff
@@ -0,0 +1,224 @@
+                http200Response.isEmpty());
+
+        Map<String, Object> contentAnnotation = (Map<String, Object>) http200Response.get("content");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has empty \"content\" property",
+                contentAnnotation.isEmpty());
+
+        Map<String, Object> contentTypeJson = (Map<String, Object>) contentAnnotation.get("application/json");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"content\" but empty \"application/json\" property",
+                contentTypeJson.isEmpty());
+
+        Map<String, Object> schema = (Map<String, Object>) contentTypeJson.get("schema");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"application/json\" but empty \"schema\" property",
+                schema.isEmpty());
+
+        Map<String, Object> schemaProperties = (Map<String, Object>) schema.get("properties");
+        Assert.assertFalse(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"schema\" but empty \"properties\" property",
+                schemaProperties.isEmpty());
+
+        Assert.assertTrue(
+                "\"/districts/{code}\" \"response\" for GET verb and HTTP status 200 has \"properties\" but missing \"alias\" property",
+                schemaProperties.containsKey("alias"));
+
+        Assert.assertTrue("", ((LinkedHashMap) schemaProperties.get("alias")).containsValue("Alias of district"));
+
+        // 2.0 The @RequestBodySchema annotation has been added to provide a shorthand mechanism to specify the schema for a request body
+        Map<String, Object> getDistrictByCode20Path = (Map<String, Object>) paths.get("/districts/{code}/v20");
+        Assert.assertFalse("\"/districts/{code20}\" property is empty", getDistrictByCode20Path.isEmpty());
+
+        Map<String, Object> patchMethod = (Map<String, Object>) getDistrictByCodePath.get("patch");
+        Assert.assertFalse("\"/districts/{code20}\" \"patch\" property is empty", patchMethod.isEmpty());
+        Assert.assertEquals("\"/districts/{code20}\" \"patch\" property should have exactly 13 keys",
+                13, patchMethod.keySet().size());
+
+        Map<String, Object> requestBody = (Map<String, Object>) patchMethod.get("requestBody");
+        Assert.assertFalse("\"/districts/{code20}\" \"requestBody\" for PATCH verb is empty", requestBody.isEmpty());
+
+        Map<String, Object> content = (Map<String, Object>) requestBody.get("content");
+        Assert.assertFalse(
+                "\"/districts/{code20}\" \"requestBody\" for PATCH verb has empty \"content\" property",
+                content.isEmpty());
+
+        Map<String, Object> contentJson = (Map<String, Object>) content.get("application/json");
```

Comment:
Based on similar variables' name this could be `patchRequestContentTypeJson`, WDYT?
In general I see that these control variables' names can be confusing and maybe it was (and is) in some different test cases, so feel free to discard this comment if you want.